files/es/web/api/batterymanager/chargingtime/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: BatteryManager.chargingTime
+slug: Web/API/BatteryManager/chargingTime
+tags:
+  - API
+  - Battery
+  - Battery API
+  - Mobile
+  - Referencia
+translation_of: Web/API/BatteryManager/chargingTime
+browser-compat: api.BatteryManager.chargingTime
+---
+{{deprecated_header}}
+
+{{APIRef("Battery API")}}
+
+Indica la cantidad de tiempo, en segundos, que faltan para que la batería esté completamente cargada.
+
+> **Nota:** Incluso si el tiempo devuelto es exacto al segundo, los navegadores los redondean a un intervalo más alto (típicamente a los 15 minutos más cercanos) por razones de privacidad.
+
+## Sintaxis
+
+```js
+var time = battery.chargingTime;
+```
+
+
+La variable `time` recibe el tiempo restante en segundos para que la batería
```

Comment:
Verificar la traducción, en inglés: _`time` is the remaining time in seconds_ y aquí dice que recibe en lugar de ser